src/core/positioning/abstractgnssreceiver.h

```diff
@@ -54,7 +54,7 @@ class AbstractGnssReceiver : public QObject
 
     QString lastError() const { return mLastError; }
 
-    virtual QList<QPair<QString, QVariant>> details() const { return {}; }
+    virtual GnssPositionDetails details() const { return {}; }
```

Comment:
Can we have a docstring here?
In particular, I noticed that we are returning system locale formatted strings, which makes this very close to "presentation layer".

Reply:
Actually, if we return strings for doubles in the QVariant field, is QVariant the correct type?